WolvenKit.App/ViewModels/Shell/ChunkViewModel.cs

```diff
@@ -58,7 +58,7 @@ public partial class ChunkViewModel : ObservableObject, ISelectableTreeViewItemM
 {
     private readonly IChunkViewmodelFactory _chunkViewmodelFactory;
     private readonly IDocumentTabViewmodelFactory _tabViewmodelFactory;
-    private readonly ILoggerService _loggerService;
+    public readonly ILoggerService _loggerService;
```

Comment:
Please don't. Get a new instance of LoggerService where needed

Reply:
I need it in the RedTreeView - can you help me out?

Reply:
Use `Locator.Current.GetService` in Views. Just look up one of the other usages to see how

Reply:
OK, thanks, learned a thing :) Fixed